pkg/agent/flowexporter/connections/connections_test.go

```diff
@@ -286,5 +299,97 @@ func TestConnectionStore_DeleteConnectionByKey(t *testing.T) {
 		assert.Nil(t, err, "DeleteConnectionByKey should return nil")
 		_, exists := connStore.GetConnByKey(*testFlowKeys[i])
 		assert.Equal(t, exists, false, "connection should be deleted in connection store")
+		checkAntreaConnectionMetrics(t, len(connStore.connections))
 	}
 }
+
+func TestConnectionStore_MetricSettingInPoll(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	metrics.InitializeConnectionMetrics()
+
+	// Reference for flow timestamp
+	refTime := time.Now()
+	// Create two flows; one is already in ConnectionStore and other one is new
+	testFlows := make([]*flowexporter.Connection, 2)
```

Comment:
do you need these test flows? We do not have to test addOrUpdateConn here. You can consider `filteredConns` returned by DumpFlows as empty and just test Prometheus metrics.

Reply:
Make sense, use a empty test flows list instead.